go.sum

```diff
@@ -306,6 +306,8 @@ github.com/modern-go/reflect2 v1.0.1/go.mod h1:bx2lNnkwVCuqBIxFjflWJWanXIb3Rllmb
 github.com/mwitkow/go-conntrack v0.0.0-20161129095857-cc309e4a2223/go.mod h1:qRWi+5nqEBWmkhHvq77mSJWrCKwh8bxhgT7d/eI7P4U=
 github.com/mypurecloud/platform-client-sdk-go/v75 v75.1.0 h1:/TJCzyHGX5p+9RncdLisk5OiAPqrSkqjhgpd+ea00Rg=
```

Comment:
Might need a `go mod tidy` There shouldn't be any references to the previous SDK version left